libs/api/domains/work-machines/src/lib/workMachines.resolver.ts

```diff
@@ -89,4 +93,32 @@ export class WorkMachinesResolver {
   ) {
     return this.workMachinesService.getWorkMachineById(user, input)
   }
+
+  @Scopes(
+    ApiScope.vinnueftirlitid,
+    ApiScope.internal,
+    ApiScope.internalProcuring,
```

Comment:
When access controling with explicit scope like `ApiScope.vinnueftirlitid` that should be sufficient and shouldn't need the `ApiScope.internal` and `ApiScope.internalProcuring`.

Reply:
Alright, will remove ApiScope.internal and ApiScope.internalProcuring

Reply:
Will have to add them back for the time being. As the ApiScope.vinnueftirlitid has been configured not entirely correctly.

Reply:
@johannaagma no as the `ApiScope.vinnueftirilitid` should be configured for all of these cases. 

`ApiScope.internalProcuring` and `ApiScope.internal` are the default scopes to use when we don't need custom delegations for some functionality. But when we use a custom scope we can configure all of this on a single scope.

Reply:
@davidkef can we fix the configuration of the `ApiScope.vinnueftirlitid` and cleanup the `internal` and `internalProcuring` scopes before merging?